sql/core/src/test/resources/sql-tests/results/group-analytics.sql.out

```diff
@@ -101,29 +101,29 @@ SELECT course, year, SUM(earnings) FROM courseSales GROUP BY ROLLUP(course, year
 -- !query schema
 struct<course:string,year:int,sum(earnings):bigint>
 -- !query output
-NULL	NULL	113000
-Java	NULL	50000
 Java	2012	20000
 Java	2013	30000
-dotNET	NULL	63000
+Java	NULL	50000
+NULL	NULL	113000
 dotNET	2012	15000
 dotNET	2013	48000
+dotNET	NULL	63000
 
 
 -- !query
 SELECT course, year, SUM(earnings) FROM courseSales GROUP BY CUBE(course, year) ORDER BY course, year
 -- !query schema
 struct<course:string,year:int,sum(earnings):bigint>
 -- !query output
-NULL	NULL	113000
-NULL	2012	35000
-NULL	2013	78000
-Java	NULL	50000
 Java	2012	20000
 Java	2013	30000
-dotNET	NULL	63000
+Java	NULL	50000
+NULL	2012	35000
```

Comment:
Well, are we sorting `NULL` as a string? This looks like hiding another kind of bugs, e.g., Sorting with NULL first.

Reply:
Ah, I see. I missed that point. I'll look for another way to avoid the case.